content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4/index.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Upgrade to Prisma 4'
+metaTitle: 'Upgrade to Prisma 4'
+metaDescription: 'Guides on how to upgrade to Prisma 4'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+Prisma 4 introduces a number of **breaking changes** if you are upgrading from an earlier version (`3.x.x`). This guide explains how this upgrade might affect your application and gives instructions on how to handle any changes.
+
+</TopBlock>
+
+## Breaking changes
+
+### Index configuration
+
+### Scalar list defaults
+
+For database connectors that support scalar lists (PostgreSQL, CockroachDB and MongoDB), Prisma 4 introduces the ability to set a default value in your Prisma schema file using the `@default` attribute:
+
+<TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
+<tab>
+
+```prisma highlight=3;normal
+model User {
+  id             Int      @id @default(autoincrement())
+  posts          Post[]
+  favoriteColors String[] @default(["red", "yellow", "purple"])
+}
+```
+
+</tab>
+<tab>
+
+```prisma highlight=3;normal
+model User {
+  id             String   @id @default(auto()) @map("_id") @db.ObjectId
+  posts          Post[]
+  favoriteColors String[] @default(["red", "yellow", "purple"])
+}
+```
+
+</tab>
+</TabbedContent>
+
+This is a breaking change if you previously had defaults defined for scalar lists at the database level.
+
+### Explicit `@unique` constraints on one-to-one relations
+
+### Removing `references` syntax for implicit many-to-many relations
```

Comment:
```suggestion
### Remove `references` syntax for implicit many-to-many relations
```

Reply:
Thanks, done. (I did a bunch at once separately from the suggestion, which had got out of date)